pkg/updater/updater.go

```diff
@@ -49,15 +49,17 @@ const componentName = "updater"
 
 // Metrics holds metrics relevant to the Updater.
 type Metrics struct {
-	UpdateState  metrics.Cyclic
-	DelaySeconds metrics.Duration
+	UpdateState       metrics.Cyclic
+	DelaySeconds      metrics.Duration
+	IncompleteUpdates metrics.Counter
 }
 
 // CreateMetrics creates metrics for this controller
 func CreateMetrics(factory metrics.Factory) *Metrics {
 	return &Metrics{
-		UpdateState:  factory.NewCyclic(componentName),
-		DelaySeconds: factory.NewDuration("delay", "Seconds updater is behind schedule", "component"),
+		UpdateState:       factory.NewCyclic(componentName),
+		DelaySeconds:      factory.NewDuration("delay", "Seconds updater is behind schedule", "component"),
+		IncompleteUpdates: factory.NewCounter("counter", "number of unread columns"),
```

Comment:
Please name this counter something more descriptive, such as "incomplete-updates".
Amend the description also, please. This counts the number of update attempts that don't complete, not the number of columns that were skipped over.

Note [here](https://github.com/GoogleCloudPlatform/testgrid/blob/f7fef68af4595f39ca416c9e3a070bc8885a5165/util/metrics/metrics.go#L48) that the arguments for NewCounter are (name, description, ...any additional fields to capture)